python/ray/client_builder.py

```diff
@@ -26,7 +26,8 @@ class ClientInfo:
 
 class ClientBuilder:
     """
-    Builder for a Ray Client connection.
+    Builder for a Ray Client connection. This class can be subclassed
+    by other builder classes.
```

Comment:
Could help to explain how/why it should be subclassed.
For example could add "to modify connection behavior" at the end of the second sentence.

Reply:
Great idea!